src/Umbraco.Web/PropertyEditors/RichTextPropertyEditor.cs

```diff
@@ -99,6 +112,11 @@ public override object FromEditor(Core.Models.Editors.ContentPropertyData editor
 
                 var editorValueWithMediaUrlsRemoved = TemplateUtilities.RemoveMediaUrlsFromTextString(editorValue.Value.ToString());
                 var parsed = MacroTagParser.FormatRichTextContentForPersistence(editorValueWithMediaUrlsRemoved);
+
+                var userId = Current.UmbracoContext.Security.CurrentUser.Id;
```

Comment:
Same issue as above, you need to at least null check UmbracoContext here since there's no guarantee this is running in a request.